### DISCUSSION TITLE
Need details about Ansible Code Linting setup.

### DISCUSSION THREAD

**Discussion by Shubhij12** [super-linter/super-linter#6278] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When setting up Ansible code linting, it is advised to create a dedicated directory for Ansible, referred to as 'ANSIBLE_DIRECTORY' variable to' /ansible'. I would like to clarify whether this directory is intended solely for ansible config files or if it can also accommodate other YAML files. 
If it can include additional files, I would like to know if it is limited to YAML formats or if other file types are permissible as well[.ps1,.py]. Additionally, I seek to understand the rationale behind establishing a separate directory for Ansible.

### Expected Behavior

Is it possible to include All Code languages in single directory?

**Comment by ferrarimarco** [comment]
Hi @Shubhij12 

I moved this to a discussion.

> I would like to clarify whether this directory is intended solely for ansible config files or if it can also accommodate other YAML files. If it can include additional files, I would like to know if it is limited to YAML formats or if other file types are permissible as well[.ps1,.py]

This directory can contain whatever you like. It's intended to be the directory where Super-linter points `ansible-lint` to. You can structure it as you prefer to contain your Ansible roles, playbooks, modules, inventories...

> I seek to understand the rationale behind establishing a separate directory for Ansible.

This is because of how `ansible-lint` works. It expects to lint either single playbooks, or an entire directory tree that respects Ansible conventions in terms of [directory layout](https://docs.ansible.com/ansible/2.8/user_guide/playbooks_best_practices.html#directory-layout).
